zerver/lib/webhooks/common.py

```diff
@@ -61,6 +63,20 @@ def __init__(self, header: str) -> None:
     def msg_format() -> str:
         return _("Missing the HTTP event header '{header}'")
 
+class UnsuccessfulResponseError(JsonableError):
```

Comment:
The name could be more descriptive here, maybe `ThirdPartyAPICallbackError`? Or maybe something similar?

Reply:
`ThirdPartyAPICallbackError` sounds pretty good, I'll make the change :)